src/annotator/sidebar.js

```diff
@@ -439,4 +440,12 @@ export default class Sidebar extends Guest {
   setAllDoodleOptions(options) {
     this.crossframe.call('setDoodleOptions', options);
   }
+
+  /**
+   * (CreativeNTR) Save the doodle
+   */
+  saveDoodle() {
+    // TODO figure out what to call here to save the doodle
```

Comment:
Extra TODO comment, we have all the crossframe calls linked up right? I'd guess so because it's actually successfully saving doodles.

Reply:
Yes! good catch